src/bindings/python/flux/wrapper.py

```diff
@@ -279,7 +279,7 @@ def __getattr__(self, name):
             raise AttributeError
 
         # try it bare
-        llib = getattr(self, "lib")
+        llib = self.lib
```

Comment:
@jameshcorbett: thanks for catching this!   This is a tricky edge-case for sure!

I'm thinking it might be helpful to check if `name == lib` and if so, `raise AttributeError`, like above.